### DISCUSSION TITLE
COLMAP using no GPU?

### DISCUSSION THREAD

**Discussion by DuVogel87** [colmap/colmap#2637] [Q&A]
Hello.
I checked colmap --help. There it says "colmap with cuda" 
But still task manager and open hardware monitor show, that only cpu is been used heavily during "matching process". gpu seems not to be used. whats the matter here?
i installed colmap 3 months ago while i was running a rtx1070ti. last week i changed my gpu to a rtx2080ti. could that be an issue somehow?

![colmap exe open hardware monitor](https://github.com/colmap/colmap/assets/123517103/9b9d4064-0e0a-4fc4-b6ae-c52202cc9259)
![colmap exe task manager](https://github.com/colmap/colmap/assets/123517103/f6b3cfd3-d35d-438e-9102-b5f0434cd455)
![colmap with cuda](https://github.com/colmap/colmap/assets/123517103/237cbd2c-8344-458d-b428-e4693ca5d885)

**Comment by luyuqiu888** [comment]
Hello, may I ask if your problem has been solved? I also encountered the same problem. I downloaded the cuda version of colmap, but when colmap was running, the cpu usage was very high and the gpu was almost useless. My nvidia graphics card is a 4060

**Comment by DuVogel87** [comment]
The gpu will only be used during feature extraction and feature matching. And also if you check it via gui or enable it via command line.

via command line you can see the help:

colmap feature_extractor —help
colmap feature_matcher — help

the“—„ stands for „- -„ but without the space.
